test/integration/user_views_past_orders_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class UserViewsPastOrdersTest < ActionDispatch::IntegrationTest
+  test "user sees all submitted orders" do
+     user = create(:user)
+     ApplicationController.any_instance.stubs(:current_user).returns(user)
+
+     order1 = add_two_items_to_cart
+     click_on "Checkout"
+     order2 = add_two_items_to_cart
+     click_on "Checkout"
+
+     visit orders_path
+
+     assert_equal 2, Order.all.count
+   end
```

Comment:
`end` at 16, 3 is not aligned with `test "user sees all submitted orders" do` at 4, 2.